base/ca/src/com/netscape/cms/profile/def/ServerKeygenUserKeyDefault.java

```diff
@@ -311,7 +311,7 @@ public void populate(IRequest request, X509CertInfo info)
 
                 EncryptionAlgorithm encryptAlgorithm =
                         EncryptionAlgorithm.AES_128_CBC_PAD;
-                KeyWrapAlgorithm wrapAlgorithm = KeyWrapAlgorithm.RSA;
+                KeyWrapAlgorithm wrapAlgorithm = KeyWrapAlgorithm.RSA_OAEP;
```

Comment:
@jmagne / @ladycfu -- is this the right approach? Do we want to fully replace PKCS#1 v1.5, or are we going to make it configurable instead?

Reply:
My take is, since we are still tackling HSM, it might be best to make it globally configurable to switch between the two.  It would actually be more convenient for development as well when there is a question of whether the code was working before switching to OAEP.

Reply:
@cipherboy @jmagne , so was this addressed?  I'm not sure I see anything. Thanks.

Reply:
@ladycfu Not yet, we were just trying to get it to build :)

Reply:
@ladycfu i t should be addressed now :)